src/libsyntax/parse/mod.rs

```diff
@@ -168,10 +168,13 @@ crate fn parse_stmt_from_source_str(name: FileName, source: String, sess: &Parse
     new_parser_from_source_str(sess, name, source).parse_stmt()
 }
 
-pub fn parse_stream_from_source_str(name: FileName, source: String, sess: &ParseSess,
-                                    override_span: Option<Span>)
-                                    -> TokenStream {
-    source_file_to_stream(sess, sess.source_map().new_source_file(name, source), override_span)
+pub fn parse_stream_from_source_str(
+    name: FileName,
+    source: String,
+    sess: &ParseSess,
+    override_span: Option<Span>,
+) -> TokenStream {
+    source_file_to_stream(sess, sess.source_map().new_source_file(name, source), override_span).0
```

Comment:
does this mean that for rustc invocations that don't read the source from a file but from stdin we don't get these suggestions?

Reply:
Correct. On the one hand, those will not have any _worse_ output. On the other, this was a bit explorative and want to nail it down before trying to get to to work in all cases.